speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/CHANGES.txt

```diff
@@ -312,13 +332,41 @@ Changed:
                     removed selectTrackMap()
 
 
-  data/data/menu/
-    mainmenu.xml
-        added a back button
-    soundconfigmenu.xml
-        added controls for intervention sound volume
-    controlconfigmenu.xml
-        added controls for intervention toggle on/off
+  data/
+      config/raceman
+        CMakeLists.txt
+        quickrace.xml
+            changed standard track
+        practice.xml
+            removed
```

Comment:
Something like this would be more clear and save space, do this for all removed files.
```suggestion
        removed practice.xml
```